release/ami/image.pkr.hcl

```diff
@@ -19,7 +19,7 @@ locals {
 # https://cloud-images.ubuntu.com/locator/ec2/
 
 source "amazon-ebs" "x86_64" {
-  ami_name      = "earthly-${var.earthly_version}-x86_64-${local.timestamp}"
+  ami_name      = "earthly-${var.earthly_version}-amd64-${local.timestamp}"
```

Comment:
Also a small fix to align the AMI names with the architecture component of our platform strings

Reply:
I already fixed the existing one by copying the AMI to match this naming pattern.